Enable optimized op_le broadcast against 1 element tensor

manuelcandales opened the pull request:
Summary: Will enable full broadcasting support in a future PR. But right now, enabling the 1-element case, to resolve  https://github.com/pytorch/executorch/issues/9433

Differential Revision: D71652593

guangy10 commented:
Cool! I have verified the fix. Thanks @manuelcandales ! Can you please fix the linter so we can merge it and then cherry-pick to the release branch. It's affecting several HF models, so would want this fix to be included in the 0.6 release. https://github.com/huggingface/optimum-executorch/issues/14

manuelcandales committed:
Enable optimized op_le broadcast against 1 element tensor (#9507)

Summary:

Will enable full broadcasting support in a future PR. But right now, enabling the 1-element case, to resolve  https://github.com/pytorch/executorch/issues/9433

Reviewed By: larryliu0820, guangy10

Differential Revision: D71652593
